packages/cli-link-assets/package.json

```diff
@@ -26,7 +26,6 @@
   ],
   "devDependencies": {
     "@react-native-community/cli-types": "14.0.0-alpha.4",
-    "@types/opentype.js": "^1.3.8",
```

Comment:
@fabioh8010 why do we need this? any case we could remove the dep altogether? (as a part of this PR its source was brought to this package and modified)

Reply:
`opentype.js` is a core library in `cli-link-assets` used to analyse the font's metadata. I added `@types/opentype.js` because `opentype.js` doesn't have any TS support.

Reply:
doesn't look like we can get rid of `opentype.js`, we could just limit the vendored types to what's actually needed, WDYT?

Reply:
Sounds good to me!